File: MODUL.2/PRAK205_2310817120010_NurHikmah.c
#include <stdio.h>
#include <math.h>

int main()
{
    int A, B;
    printf("Input\n");
    scanf("%d %d", &A, &B);

    int C = sqrt(pow(B, 2) - (pow(A, 2)));
    int keliling = A + B + C;
    int luas = (C * A) / 2;

    printf("\nOutput\n");
    printf("Alas        = %d cm \n", C);
    printf("Tinggi      = %d cm \n", A);
    printf("Keliling    = %d cm \n", keliling);
    printf("Luas        = %d cm^2 \n", luas);

    return 0;
}

File: MODUL.2/PRAK205_2310817120010_NurHikmah.py
import math

print('Input')
nilai = input()

angka = nilai.split()
A, B = map(int, angka)

C = int(math.sqrt(B ** 2 - A ** 2)) 
keliling = int(A + B + C)
luas = int(1 / 2 * C * A)

print('\nOutput')
print(f'Alas        = {C} cm')
print(f'Tinggi      = {A} cm')
print(f'Keliling    = {keliling} cm')
print(f'Luas        = {luas} cm^2')
